scripts/readme.md

```diff
@@ -0,0 +1,10 @@
+# Generate data
+
+# Generate links
```

Comment:
Thanks for documenting. Does the script just console out the new values or do you need to document where it stores the values?

Do we need to document in a deployment document for the platform how to do this generation?